sortfilterproxymodel.qbs

```diff
@@ -70,7 +73,47 @@ Product
         [
             "filter.h",
             "qqmlsortfilterproxymodel.h",
+            "qqmlsortfilterproxymodelexport.h",
+            "qqmlsortfilterproxymodel_plugin.h",
             "sorter.h"
         ]
-    }    
+    }
+
+    Group
+    {
+        name: "Other files"
+        files:
+        [
+            "qmldir"
+        ]
+        fileTags: "qbs resources"
+    }
+
+    //FIXME: Replace this to actually install files in the right location
```

Comment:
We have to create an issue in quarks/avalon/etc.. to actually "install" files, instead of using the 
destinationDirectory: project.deployDirectory . So this is to be cleaned up then.